internal/dao/dao_interfaces.go

```diff
@@ -74,7 +74,7 @@ type ReservationDao interface {
 
 	// ListInstances returns instances associated to a reservation. UNSCOPED.
 	// It currently lists all instances and not instances for a reservation, this is a TODO.
-	ListInstances(ctx context.Context, limit, offset int64) ([]*models.ReservationInstance, error)
+	ListInstances(ctx context.Context, reservationId int64) ([]*models.ReservationInstance, error)
```

Comment:
Just a note: this function was unused and it was badly designed - it was meant to return instances for particular reservation, not all instances (that makes no sense).

Also the patch enforces account ID, so it can only be used with proper account context.